### ISSUE TITLE
Highlight block not working properly.

### ISSUE THREAD

**Issue by 1Ox1O** [mrkite/TerraFirma#129]
Hello, I've been very happy with how Terrafirma worked so far, and happened to stumble upon this by complete luck.
I would describe it but the video I took would do that much faster: https://streamable.com/udcl5t

**Comment by mrkite** [comment]
Yeah, blocks under water (and lava) don't highlight properly.  I'm not
entirely sure how best to fix it.
